roles/auth/templates/config.json.j2

```diff
@@ -26,7 +26,7 @@
     "signinConfirmation": {
       "enabled": true,
       "sample_rate": 0,
-      "forceEmailRegex": ["^sync.*@restmail.net$", "@mozilla.com$"]
+      "enabledEmailAddresses": /.+@mozilla\.com$/
```

Comment:
So a couple of notes:

1. If we want to preserve legal JSON, it will parse from a string to the `RegExp` instance so this could also be `".+@mozilla\\.com$"`.

2. If we want to retain both addresses in the `RegExp`, something along the lines of `"^(sync.*@restmail\\.net)|(.+@mozilla\\.com)$"` should be okay, right?